fem/fe/fe_nurbs.hpp

```diff
@@ -56,6 +56,11 @@ class NURBSFiniteElement : public ScalarFiniteElement
    Vector              &Weights    ()         const { return weights; }
    /// Update the NURBSFiniteElement according to the currently set knot vectors
    virtual void         SetOrder   ()         const { }
+
+   void GetIJK(int *IJK) const
```

Comment:
This could easily lead to segmentation faults but it is consistent with the API of this class. At the very least I think we need to document this method and make the expected length of `IJK` clear.